integration-libs/cdc/schematics/add-cdc/schema.json

```diff
@@ -25,6 +25,23 @@
       "type": "array",
       "uniqueItems": true,
       "default": ["CDC"]
+    },
+    "jsSDKUrl": {
+      "type": "string",
+      "description": "Location of the CDC Java Script SDK in Customer Data Cloud",
+      "x-prompt": "[CDC] What is the CDC JS SDK URL?"
+    },
+    "baseSite": {
+      "type": "string",
+      "description": "Name of the Spartcus base site",
+      "x-prompt": "[CDC] What is the base site? (optional)",
+      "default": "electronics-spa"
+    },
+    "sessionExpiration": {
+      "type": "number",
+      "description": "Session Expiration time in seconds",
+      "x-prompt": "[CDC] What is the CDC Session expiration time (in seconds)? (optional)",
```

Comment:
To make things clearer, I believe these are all required than optional by looking at the 'old' config.

If that's the case, let us omit (optional) and place the default property for these three CLI params as it was in statically set in the config before.

Reason why I'm saying I'm thinking they are NOT optional is because if they are indeed optional, we should be omitting the creation of these properties in the config. For example as this was used as an example when we spoke in terms of what CDS/ISS did, you would need to create a different config for these different scenarios. https://github.com/SAP/spartacus/blob/develop/integration-libs/cds/src/schematics/add-cds/index.ts#L45-L87

Reply:
Hi @bgambocjaviniar,

thanks for the review.

However, the `jsSDKUrl` is not optional and it is required to be given by the customer.
Unless the `jsSDKUrl` is not provided, the entire CDC functionality does not work.

Also, we do not have a default URL that can be used if the user does not provide any, since it would point to our dev. CDC instance. 

This was the reason for keeping `jsSDKUrl`mandatory.

Even if we have multiple configurations, we might not be able to expose our dev. CDC instances.
The other 2 configurations like `baseSite` and `sessionExpiration` have fallback default values which can be used, so they were mentioned as optional.


Please let me know your thoughts.

Reply:
I believe I already explained my thoughts. 

To simplify things
- all your options are required due to the fact that you have default values in case the user presses 'enter'. Meaning we still want to have those required configs to be placed in the file.
- Yes jsSDKUrl is not optional and I did not say anything about it being an issue, but nothing stops the user from pressing enter. Again, we would need a default value placed it in as mentioned in other instances below.

It is just a matter of following how 'CDS' constructed it as you pointed out somewhere in a slack thread.

Reply:
I see that you took a look at the rest of the comments https://github.com/SAP/spartacus/pull/15756#discussion_r876555817